src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private String name = "circle";
```

Comment:
There is no need for this field for several reasons:
1. If you want to know figure name just call getClass() on figure instance
2. This field is common for all figures and should be on abstract class level, but there are requirement that only one field (color) should be common

Reply:
thx for help, fixed